parsons/databases/redshift/redshift.py

```diff
@@ -586,6 +588,9 @@ def copy(
             aws_secret_access_key:
                 An AWS secret access key granted to the bucket where the file is located. Not
                 required if keys are stored as environmental variables.
+            aws_session_token:
+                An AWS session token granted to the bucket where the file is located. Not
+                required if keys are stored as environmental variables.
```

Comment:
Can you give the exact environmental variable the session token needs to be stored in as part of the documentation?